Jenkinsfile

```diff
@@ -308,6 +309,56 @@ def buildDiffCoverage() {
     }
 }
 
+def buildPerformance() {
+  return {
+    // Select docker-cph-X.  We want docker, metal (brix) and only one executor
+    // (exclusive), if the machine changes also change REALM_BENCH_MACHID below
+    node('docker && brix && exclusive') {
+      getSourceArchive()
+
+      def gitTag = readGitTag()
+      def gitSha = readGitSha()
```

Comment:
This is the second instance, the first being on [line 16](https://github.com/realm/realm-core/blob/e106b520f930430143f143139008fc3f381e2655/Jenkinsfile#L16), no?